openshift/templates/backup/config/backup.conf

```diff
@@ -47,10 +47,8 @@
 # 0 4 * * * default ./backup.sh -s -v all
 # ============================================================
 postgres=vc-authn-database/vc_authn
-postgres=vc-authn-wallet/vc_authn_agent_wallet
 postgres=wallet-lsbc/agent_lsbc_wallet
 postgres=wallet-prime/agent_prime_wallet
-postgres=wallet-prime/agent_sandbox_prime_wallet
```

Comment:
Where did the sandbox agent backup go?

Reply:
sorry, I must have removed it when I was backing up prod and test